src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -87,12 +92,18 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
    * Currently selected phase split
    */
   selectedPhaseSplit: any = null;
+  selectedPhaseSplitLeaderboardToggle: boolean = null;
```

Comment:
@drepram I think you need to initialize it with the value which is fetched from the back-end and also please change this variable name to `showLeaderboardByLatest`.

Reply:
Like I said before, what if the variable is not present in the objects retrieved from the back-end. Please read my previous messages on our previous conversation here.

Reply:
This will not be the case, there will always be this field with the default value as `False`.

Reply:
Changed the name.